Jenkinsfile

```diff
@@ -24,6 +24,7 @@ elifePipeline {
                         builderSmokeTests 'search--end2end', '/srv/search'
                         builderCmd 'search--end2end', 'cd /srv/search; php bin/console queue:import all --env=end2end'
                         builderCmd 'search--end2end', 'cd /srv/search; ./bin/wait-for-empty-queues end2end'
+                        builderCmd 'search--end2end', "cd /srv/search; ./bin/reindex end2end elife_search_${env.BUILD_NUMBER}"
```

Comment:
This is a testing environment, so we were attempting to test both an import from scratch and a reindexing. But the import from scratch is not cleaning the db first, so it should do that. Then we could change the order: 1. update the existing index, then 2. test also the import from scratch.